apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable react/prop-types */
-// issues are validated using correct shape - eslint problem?
```

Comment:
I don't like that we're disabling this for the entire file, especially such a large file as this. What problem was this working around?

Reply:
Oh, nevermind, you removed this. Good! 😄